app/helpers/billing_helper.rb

```diff
@@ -0,0 +1,17 @@
+module BillingHelper
+  private
+
+  def default_plan?(account)
+    installation_config = InstallationConfig.find_by(name: 'CHATWOOT_CLOUD_PLANS')
+    default_plan = installation_config.value.first
+    account.custom_attributes['plan_name'] == default_plan['name']
+  end
+
+  def conversations_this_month(account)
+    account.conversations.where('created_at > ?', Time.zone.now.beginning_of_month).count
```

Comment:
We are not following calendar date billing. The period is calculated based on the date they signed up. The desired implementation would the number from start of the billing period to current date. I think taking `30.days ago` is also fine for the start.